src/moduler/aktivitet/visning/hjelpekomponenter/AktivitetinformasjonVisning.tsx

```diff
@@ -63,17 +66,18 @@ const AktivitetinformasjonVisning = (props: Props) => {
                     </InternLenke>
                 </div>
                 <VisningIngress aktivitetstype={type} />
-                <AvtaltMarkering hidden={!avtalt} className="aktivitetvisning__etikett" />
-                <IkkeDeltMarkering visible={ikkeDelt} className="aktivitetvisning__etikett" />
+                <AvtaltMarkering hidden={!avtalt} className={styles.etikett} />
+                <IkkeDeltMarkering visible={ikkeDelt} className={styles.etikett} />
             </div>
             <AvtaltContainerNy
                 underOppfolging={underOppfolging}
                 aktivitet={valgtAktivitet}
-                className="aktivitetvisning__underseksjon"
+                className={styles.underseksjon}
             />
-            <div className="aktivitetvisning__underseksjon">
+            <div className={styles.underseksjon}>
                 <Aktivitetsdetaljer valgtAktivitet={valgtAktivitet} />
             </div>
+            {valgtAktivitet.type === STILLING_FRA_NAV_TYPE ? <MeldInteresseForStillingen /> : null}
```

Comment:
Tror dette kan skrives sånn her. Evt kan også komponenten selv styre om den vises eller ikke
```suggestion
            {valgtAktivitet.type === STILLING_FRA_NAV_TYPE && <MeldInteresseForStillingen />}
```

Reply:
👍